src/data.ts

```diff
@@ -230,8 +230,9 @@ export const scheduleSaturday: cusecEvent[] = [
   {
     startTime: { hour: 12, minute: 15 },
     endTime: { hour: 13, minute: 15 },
-    title: "Workshop by RBC",
-    description: "Analyzing a day in my Spotify life with technology",
+    title: "Andrew Briggs - Technical Lead at RBC",
+    description:
+      "Practicing Architecture by analyzing a day in my Spotify life. Please read [GitHub repo](https://github.com/atbriggs/nsync) prior to Workshop!",
```

Comment:
Andrew just gave me feedback, which is actually pretty good about the word `read`. Perhaps it's not specific enough 🤔  since we'd also like students to download everything prior to be nice and ready. 

@sttic what do you think about changing it to
```
Please download the pre-requisites and try to run the apps prior to the workshop if you'd like to participate. Instructions can be found in the [GitHub repo](https://github.com/atbriggs/nsync).
```

Reply:
I agree! I'll update the description to what you have there